libsignal-service/src/receiver.rs

```diff
@@ -44,10 +45,15 @@ impl<Service: PushService> MessageReceiver<Service> {
     pub async fn create_message_pipe(
         &mut self,
         credentials: ServiceCredentials,
+        allow_stories: bool,
     ) -> Result<MessagePipe, ServiceError> {
+        let headers = &[(
```

Comment:
since `HeaderName::from_static` is `const fn` maybe we could instead store it in a constant (same for the value?)

Reply:
Looks like `actix_web` re-exports `http` (https://docs.rs/actix-http/3.4.0/src/actix_http/header/map.rs.html#6) so it should be good to share the type. The `http` crate version also shouldn't be an issue at it's been _very_ stable.

Reply:
Honestly, if we're going there, I think we should just get rid of the split altogether and get Whisperfish to run on the Hyper backend. That would get rid of a lot more :bullettrain_front: :hankey: at the same time.

Reply:
Fair point, feel free to merge as-is, but maybe having a single backend would also let us clean things up a little bit (not having an `async trait` for `PushService` and maybe a bit more magic as well). If you think it's the way to go, we can definitely do that.

Reply:
It's on my list of long-term things to get in Whisperfish. If we move to hyper some day, I'll certainly come back to that.